docs/kubernetes-shortguide-definitions/calico-definition-shortguide/index.md

```diff
@@ -16,4 +15,4 @@ show_on_rss_feed: false
 
 ### Calico
 
-[Calico](https://docs.projectcalico.org/v2.0/getting-started/kubernetes/) enables networking, and the networking policy through the NetworkPolicy API to provide simple, scalable, virtual networking. Calico uses BGP to distribute routes for Kubernetes Pods without the need for overlays.
+[Calico](https://docs.projectcalico.org/v3.8/introduction/) is an implementation of Kubernetes' networking model, and it served as the original reference for the Kubernetes [NetworkPolicy API](https://kubernetes.io/docs/concepts/services-networking/network-policies/) during its development. Calico also provides advanced policy enforcement capabilities that extend beyond Kubernetes' NetworkPolicy API, and these can be used by administrators alongside that API. Calico uses BGP to distribute routes for Kubernetes Pods, without the need for overlays.
```

Comment:
Something about "enables networking policy through the NetworkPolicy API" didn't feel like it told the whole story here, so I expanded this a bit.